usagov/usagov-2021
# USAGOV-1803-input-focus-color: Apply checked class to focused wizard radios

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1803

## Description

Update _wizard.scss to add the same styling to wizard radio buttons in the focused state as get applied on click. Also prevent users from "checking" multiple radios by clicking on a button and then moving focus to another radio. 

## Type of Changes

- [ X ] Frontend (Twig, Sass, JS)
<img width="566" alt="image" src="https://github.com/user-attachments/assets/dbdf1b16-7bca-47c2-96f6-c386a99568da">

## Testing Instructions


1. Navigate to a wizard page with options.
2. Using your keyboard, tab into the radio selection area.
3. Select the first radio this way, or use the right and left keyboard arrows to select a specific radio.
4. Using your mouse, click on a different radio option.
5. The radio option selected in step 2/3 is unselected, and the new radio option is selected.
6. Use your right or left keys to change your selection.
7. The radio option in step 5 is unselected and the new option is selected as expected.
8. Click on the "next" button.
9. You are brought to the next page in the wizard as expected.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
10. Find the commit of this pull request.
11. Build and deploy the changes.
12. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
